core/ide/src/main/java/org/alice/ide/Theme.java

```diff
@@ -54,9 +54,14 @@
  * @author Dennis Cosgrove
  */
 public interface Theme {
-  public static final Dimension DEFAULT_SMALLER_ICON_SIZE = new Dimension(32, 24);
-  public static final Dimension DEFAULT_SMALL_ICON_SIZE = new Dimension(40, 30);
-  public static final Dimension DEFAULT_LARGE_ICON_SIZE = new Dimension(120, 90);
+  public static final Dimension EXTRA_SMALL_RECT_ICON_SIZE = new Dimension(32, 24);
+  public static final Dimension SMALL_RECT_ICON_SIZE = new Dimension(40, 30);
```

Comment:
The size descriptors were already here, but with the new sizes added this small and extra small look closer to medium and small in square. Consider renaming these two up to small and medium.
Could go another step and add a new extra small at the 24x16 to be used in ResourceNodeTreeState